### ISSUE TITLE
Is there a simple way to split the endpoints of an application in two different openapi-files?

### ISSUE THREAD

**Issue by sigmanil** [papsign/Ktor-OpenAPI-Generator#124]
Is there a simple way to split the endpoints of an application in two different openapi-files? We have some endpoints that are intended for internal use and some for external use.

**Comment by Wicpar** [comment]
No, the config is unfortunately global, but it would be a nice addition
